docs/DeveloperGuide.md

```diff
@@ -554,3 +554,111 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 * **Mainstream OS**: Windows, Linux, Unix, OS-X
 * **Private contact detail**: A contact detail that is not meant to be shared with others
+
+
+## **Appendix: Instructions for manual testing**
+
+###Interview Management
+
+####Add an Interview
+
+1. Adding an interview without any candidates to HR Manager<br>
+   i. Prerequisites: Position exists in HR Manager.
+   ii. Test case: `add_i position=Bookkeeper date=15/10/2021 time=1400 duration=120 interviewed=pending`<br>
+   Expected: new interview for `Bookkeeper` on `15 Oct 2021`, `1400 ~ 1600` with status `PENDING` is added to the bottom of the list of interviews.<br>
+
+2. Adding an interview with first candidate to HR Manager<br>
+   i. Prerequisites: Position exits in HR Manager, current displayed list of candidates is not empty and candidate with corresponding index has applied for position.<br>
+   ii. Test case: `add_i position=Bookkeeper c=1 date=15/10/2021 time=1400 duration=120 interviewed=pending`<br>
+   Expected: new interview for `Bookkeeper` on `15 Oct 2021`, `1400 ~ 1600` with status `PENDING`, with first candidate from the currently displayed list of candidates is added to the bottom of the list of interviews.<br>
+
+3. Adding an interview with multiple candidates to HR Manager<br>
+   i. Prerequisites: Position exits in HR Manager, current displayed list of candidates has more than 1 entry and candidates with corresponding indexes has applied for position.<br>
+   ii. Test case: `add_i position=Bookkeeper c=1 2 date=15/10/2021 time=1400 duration=120 interviewed=pending`<br>
```

Comment:
Won't this test case conflict test case 2? Cause they are the same interview right?

Reply:
I was thinking that they are alternative test cases but sure I'll change it